contracts/platform/include/platform/platform.hpp

```diff
@@ -59,6 +58,30 @@ using game_table = eosio::multi_index<
                    >;
 
 
+static uint64_t get_token_pk(const std::string& token_name) {
```

Comment:
const?

Reply:
this is a free function, `const` is a modificator applied to member functions of a class.